pkg/scaffold/operator.go

```diff
@@ -61,6 +61,10 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+			- name: POD_NAME
```

Comment:
Could you just fix the indentation here? It appears there are tabs here vs spaces everywhere else, which explains the strange way it looks.